tests/loadtests/vkloadtests.cmake

```diff
@@ -26,13 +26,22 @@ if(APPLE)
         set( MOLTEN_VK_FRAMEWORK ${VULKAN_SDK}/iOS/framework/MoltenVK.framework )
         if( NOT IS_DIRECTORY ${MOLTEN_VK_FRAMEWORK})
             # Fallback: Older Vulkan SDKs have MoltenVK.framework at a different sub path
-            message("Could not find MoltenVK.framework (at VULKAN_SDK dir '${VULKAN_SDK}')")
+            message("Could not find MoltenVK framework (at '${MOLTEN_VK_FRAMEWORK}')")
             set(MOLTEN_VK_FRAMEWORK ${VULKAN_SDK}/iOS/MoltenVK.framework)
         endif()
         if( NOT IS_DIRECTORY ${MOLTEN_VK_FRAMEWORK})
-            message(SEND_ERROR "Could not find MoltenVK.framework (at VULKAN_SDK dir '${VULKAN_SDK}')")
+            message("Could not find MoltenVK framework (at '${MOLTEN_VK_FRAMEWORK}')")
+            # Fallback: One newer Vulkan SDKs it's a .xcframework at the root level
+            # CMake does not support linking those directly (see https://gitlab.kitware.com/cmake/cmake/-/issues/21752),
+            # so we manually pick the static library file for iOS arm64 from a subfolder here
+            if( IS_DIRECTORY ${VULKAN_SDK}/MoltenVK.xcframework )
+                set( MOLTEN_VK_FRAMEWORK ${VULKAN_SDK}/MoltenVK.xcframework/ios-arm64/libMoltenVK.a )
+            endif()
+        endif()
+        if( NOT EXISTS ${MOLTEN_VK_FRAMEWORK})
+            message(SEND_ERROR "Could not find MoltenVK framework (at VULKAN_SDK dir '${VULKAN_SDK}')")
         else()
-            message("Found MoltenVK.framwork at ${MOLTEN_VK_FRAMEWORK}")
+            message("Found MoltenVK framwork at ${MOLTEN_VK_FRAMEWORK}")
```

Comment:
Fix the typo "fram"work. I think the . is correct as `MoltenVK.framework` is the name of the package on disk.

Reply:
Will do!
I deliberately removed the dot to make it more generic (newer versions deliver `MoltenVK.xcframework`)

Reply:
Okay. Just fix the typo then.